CONFIGURE.md

```diff
@@ -18,10 +18,16 @@ osmosis --read-pbf greater-london-latest.osm.pbf \
   user="openstreetmap" password="" validateSchemaVersion="no"
 ```
 
-Loading an apidb database with Osmosis is about **twenty** times slower than loading the equivalent data with osm2pgsql into a rendering database. [``--log-progress``](https://wiki.openstreetmap.org/wiki/Osmosis/Detailed_Usage#--log-progress_.28--lp.29) may be desirable for status updates.
+ ```
+ osmosis --read-xml local_area.osm --write-apidb database=openstreetmap validateSchemaVersion=no
+ ```
+
+Loading an apidb database with Osmosis is about **twenty** times slower than loading the equivalent data with osm2pgsql into a rendering database. [``--log-progress``](https://wiki.openstreetmap.org/wiki/Osmosis/Detailed_Usage#--log-progress_.28--lp.29) may be desirable for status updates. You could improve speed by [temporarily disabling index updating](https://fle.github.io/temporarily-disable-all-indexes-of-a-postgresql-table.html) and reindex all tables after the import.
```

Comment:
Have you tried out to "temporarily disabling index updating" and seen any difference in runtime? If not, I probably wouldn't this kind of recommendation here.

I believe osmosis is doing lots of single inserts and updates rather than bulk operations, which should be the main reason for its poor performance. As osmosis is mostly unmaintained these days, it's fairly unlikely to see a complete rewrite of the importer to speed things up.